install.sh

```diff
@@ -833,28 +833,28 @@ EOABORT
   )"
 fi
 
-USABLE_GIT="$(command -v git)"
-if [[ -z "${USABLE_GIT}" ]]
+USABLE_GIT=/usr/bin/git
+if [[ -n "${HOMEBREW_ON_LINUX-}" ]]
 then
-  abort "$(
-    cat <<EOABORT
-You must install Git before installing Homebrew. See:
-  ${tty_underline}https://docs.brew.sh/Installation${tty_reset}
+  USABLE_GIT="$(find_tool git)"
+  if [[ -z "$(command -v git)" ]]
```

Comment:
What is the point of using `find_tool` if you `abort` on the next line depending on the output of `command -v`?

Reply:
Yea the code is a bit of a mess sorry. I shuffled it around until it worked :sweat_smile: 

It is used later here

https://github.com/Homebrew/install/blob/716a1d024f32890ef75ea82c18a769abc24e9475/install.sh#L849

The code would be more readable if this line was moved to 848.